core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/internal/CarbonCoreActivator.java

```diff
@@ -69,6 +70,28 @@ public void start(BundleContext context) throws Exception {
                  System.getProperty("user.language") + "-" + System.getProperty("user.country") +
                  ", " + System.getProperty("user.timezone"));
 
+        // Find the wum update level and log it.
```

Comment:
Shall we move this to a separate private method? So the information logging section will have simple code lines.

Reply:
Will update

Reply:
Moved to a new method.